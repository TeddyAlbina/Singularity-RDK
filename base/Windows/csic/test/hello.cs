public class Hello {
	static Hello() {
		System.Console.WriteLine("creating Hello");
	}
	public void Goodbye() {
		System.Console.WriteLine("Goodbye");
	}
	public static void Main() {
		System.Console.WriteLine("Hello, world");
	}
}
