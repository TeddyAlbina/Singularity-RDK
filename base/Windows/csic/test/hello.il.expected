.assembly 'hello' { }
.class public 'Hello'
	extends [mscorlib]System.Object
{
.method hidebysig specialname rtspecialname static private void .cctor() {
ldstr "creating Hello"
call  void [mscorlib]System.Console::'WriteLine'(string)
ret
.maxstack 1
} // end of constructor Hello.Hello
.method hidebysig public instance void 'Goodbye'() {
ldstr "Goodbye"
call  void [mscorlib]System.Console::'WriteLine'(string)
ret
.maxstack 1
} // end of method Hello.Goodbye
.method hidebysig public static void 'Main'() {
.entrypoint
ldstr "Hello, world"
call  void [mscorlib]System.Console::'WriteLine'(string)
ret
.maxstack 1
} // end of method Hello.Main
.method hidebysig specialname rtspecialname public void .ctor() {
ldarg.0
call instance void [mscorlib]System.Object::.ctor()
ret
.maxstack 1
} // end of constructor Hello.Hello
} // end of class Hello
